apps/cli/src/index.ts

```diff
@@ -167,21 +168,36 @@ program
     }
 
     log(`[${++step}] Collating files to deploy...`);
-    const { stdout, stderr } = await exec(`git ls-files`);
-    if (stderr) {
-      console.error(stderr);
+    const workingDir = options.workingDir || process.cwd();
+
+    let pathsToCompress: string[] = [];
+    try {
+      const { stdout, stderr } = await exec(`cd ${workingDir} && git ls-files`);
+      if (stderr) {
+        console.error("Failed to use Git list your files.");
+        console.error(stderr);
+        process.exit(1);
+      }
+
+      pathsToCompress = stdout
+        .split("\n")
+        .filter((path) => !!path && !path.endsWith(".gitignore"));
```

Comment:
Done.

Reply:
Actually I now remember that I filtered this out because it was somehow interfering with the tar-ing process. While I currently don't remember the exact reason, I did try to filter it today again but it caused issues, so I [reverted it](https://github.com/onmetal-dev/metal/pull/21/commits/8cfab1829e99342c06d4fcfc8113ef36ff1bf8f1). I'll take another swing at it later.

Reply:
```suggestion
        .filter((path) => !!path);
```

Reply:
^